Added 'dig' to list of 'other' features

jtuliani opened the pull request:
Added 'dig' to list of 'other' features (since support has now been added, see comments thread at end of page).

azurecla commented:
Hi __@fjaguero__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

Commit:
Added 'dig' to list of 'other' features

Added 'dig' to list of 'other' features (since support has now been added, see comments thread at end of page).
